app/domain/authentication/authn_jwt/consts.rb

```diff
@@ -28,10 +28,10 @@ module AuthnJwt
     CACHE_REFRESHES_PER_INTERVAL = 10
     CACHE_RATE_LIMIT_INTERVAL = 300
     CACHE_MAX_CONCURRENT_REQUESTS = 3
-    MANDATORY_CLAIMS = [EXP_CLAIM_NAME].freeze
+    ENFORCED_CLAIMS = [EXP_CLAIM_NAME].freeze
     OPTIONAL_CLAIMS = [ISS_CLAIM_NAME, NBF_CLAIM_NAME, IAT_CLAIM_NAME].freeze
     VALID_CLAIM_NAME_REGEX = /^[a-zA-Z|$|_][a-zA-Z|$|_|0-9]*$/.freeze # starts with letter $ or _ can contains digits
-    MANDATORY_CLAIMS_DENY_LIST = [ISS_CLAIM_NAME, EXP_CLAIM_NAME, NBF_CLAIM_NAME, IAT_CLAIM_NAME, JTI_CLAIM_NAME, AUD_CLAIM_NAME].freeze
+    ENFORCED_CLAIMS_DENY_LIST = [ISS_CLAIM_NAME, EXP_CLAIM_NAME, NBF_CLAIM_NAME, IAT_CLAIM_NAME, JTI_CLAIM_NAME, AUD_CLAIM_NAME].freeze
```

Comment:
```suggestion
    CLAIMS_DENY_LIST = [ISS_CLAIM_NAME, EXP_CLAIM_NAME, NBF_CLAIM_NAME, IAT_CLAIM_NAME, JTI_CLAIM_NAME, AUD_CLAIM_NAME].freeze
```
Mapping is also using the value